hp-ux/generate_wazuh_packages.sh

```diff
@@ -8,16 +8,21 @@
 
 install_path="/var/ossec"
 current_path=`pwd`
+build_tools_path="/home/okkam"
```

Comment:
Probably this directory change on each instance deploy.

Reply:
You are right. However, the GCC was built in this location so it has to be replicated in order to run GCC properly. Here you can see how it is hardcoded in the pre-build GCC:
```
# gcc -v
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/home/zhxzx/gcc94_prefix/bin/../libexec/gcc/ia64-hp-hpux11.31/9.4.0/lto-wrapper
Target: ia64-hp-hpux11.31
Configured with: ../configure --enable-nls --with-included-gettext --enable-shared --with-gnu-as --with-as=/home/okkam/bootstrap-gcc/gcc94_prefix/ia64-hp-hpux11.31/bin/as ac_cv_prog_OBJCOPY=/home/okkam/bootstrap-gcc/gcc94_prefix/bin/gobjcopy ac_cv_prog_OBJDUMP=/home/okkam/bootstrap-gcc/gcc94_prefix/bin/gobjdump --with-gmp=/home/okkam/bootstrap-gcc/gmp612_prefix --with-mpc=/home/okkam/bootstrap-gcc/mpc103_prefix --with-mpfr=/home/okkam/bootstrap-gcc/mpfr316_prefix --with-gmp-ldflags=-Wl,+b,/home/okkam/bootstrap-gcc/gmp612_prefix/lib --with-mpc-ldflags=-Wl,+b,/home/okkam/bootstrap-gcc/mpc103_prefix/lib --with-mpfr-ldflags='-Wl,+b,/home/okkam/bootstrap-gcc/mpfr316_prefix}/lib' --enable-languages=c,c++ --disable-libgomp --enable-comdat --with-dwarf2 --with-gxx-include-dir=/home/okkam/bootstrap-gcc/gcc94_prefix/include/c++ --with-local-prefix=/home/okkam/bootstrap-gcc/gcc94_prefix --prefix=/home/okkam/bootstrap-gcc/gcc94_prefix --disable-bootstrap
Thread model: posix
gcc version 9.4.0 (GCC)
```

Reply:
If I understand well, this directory is **created** and then used for GCC building, right?
In that case, why it's necessary to use `/home/`? Why not `/tmp/`?

Reply:
After discussion with @vikman90 and @chemamartinez, we conclude:
- This PR needs to be merged asap. 
- The best approach should be to re-compile the pre-built GCC in a "generic" folder instead of /home/okkam
- Although this hardcoded working folder is not the best option, the need of merging this development drives us to merge asap. 
- @chemamartinez will open an issue to modify this in the future, using `/tmp/` or another folder to build GCC.